docs/cloud/cloud-troubleshooting/issue-creating-database.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Creating database issues
```

Comment:
As mentioned below, each issue gets its own page which makes them easier to find in Nav and in search.

The YAML title should relate to the issue, e.g.,

```
title: Service stopped issue
```